lib/ruby-stackoverflow/client/response_data.rb

```diff
@@ -1,14 +1,19 @@
 module RubyStackoverflow
   class Client
     class ResponseData
-      attr_reader :data, :has_more, :error
+      attr_reader :data, :has_more, :error, :raw_response, :raw_data, :quota_max, :quota_remaining, :backoff
```

Comment:
raw_data:
I want do data to json. but couldn't.
It was possible.
```
JSON.generate(res.raw_data)
```
raw_response:
It was possible to "full response" to json.